cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -0,0 +1,221 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	goflag "flag"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/expfmt"
+	"github.com/spf13/pflag"
+)
+
+// Initialize the prometheus instrumentation and client related flags.
+var (
+	listenAddress		 string
+	metricsPath			 string
```

Comment:
I ran hack/verify-golint.sh and it passes successfully, so it seems like it should be ok. The convention for the spacing inside a var block doesn't seem to be fixed as this kind of spacing has been done at some places (for eg. https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/rbac/rest/storage_rbac.go#L80, https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/rbac/register.go#L40) and differently at other places (eg. https://github.com/kubernetes/kubernetes/blob/master/pkg/client/metrics/metrics.go#L39). I might be missing some style guide rule here, please let me know if so :)